hw4/01-routing.js

```diff
@@ -40,8 +40,44 @@ app.get('/', (req, res) => {
   res.end();
 });
 
-app.get('/welcome', (req, res) => {});
+app.get('/welcome', (req, res) => {
+  res.status(200);
+  res.set({ 'Content-Type': 'text/html' });
+  res.send(`<h1>Welcome, have fun!</h1>`);
+  res.end();
```

Comment:
No need to use end() after send(). The function send() is a combined write and end.